packages/bpk-docs/src/pages/ZIndexesPage/ZIndexesPage.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import TOKENS from 'bpk-tokens/tokens/base.raw.json';
+import IOS_TOKENS from 'bpk-tokens/tokens/ios/base.raw.json';
+import ANDROID_TOKENS from 'bpk-tokens/tokens/android/base.raw.json';
+
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import { getPlatformTokens } from './../../helpers/tokens-helper';
+
+const ZIndexesPage = () => <DocsPageBuilder
+  title="Z-Indexes"
+  tokenMap={getPlatformTokens(TOKENS, IOS_TOKENS, ANDROID_TOKENS, ({ category }) => category === 'z-indices')}
```

Comment:
@shaundon indices or indexes? 😄

Reply:
Ha so I stumbled across this http://grammarist.com/usage/indexes-indices/.

Indexes sounds better - nobody says indices 😂 .

However we cannot change the token category which unfortunately is called indices.

Reply:
Yeah I just googled it and apparently both are good :P